apis/logging/v1/clusterlogging_types.go

```diff
@@ -125,6 +126,9 @@ type LogStoreSpec struct {
 	// Specification of the Elasticsearch Log Store component
 	ElasticsearchSpec `json:"elasticsearch,omitempty"`
 
+	// Specification of the Loki Log Store component
+	LokiSpec `json:"loki,omitempty"`
```

Comment:
I would like to suggest to rename this to `LokiStackSpec` to indicate that it talks to our loki-operator managed LokiStack in contrast the ClusterLogForwarder CRD defines a LokiSpec that is dedicated to talking to third-party Loki installations. This spec is more bound to the terminology introduced by the LokiStack CRD rather than from Loki although there is an overlap, e.g. Loki likes to talk about users, we talk about tenants. Later is the correct wording but the `users` is still manifested in Loki speak for legacy reasons.

WDYT? (cc @alanconway )